Directory.Packages.props

```diff
@@ -1,46 +1,75 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <RuntimeVersion>8.0.0</RuntimeVersion>
-    <AspNetCoreVersion>8.0.7</AspNetCoreVersion>
-    <EfCoreVersion>8.0.7</EfCoreVersion>
+    <RuntimeVersion8>8.0.0</RuntimeVersion8>
+    <AspNetCoreVersion8>8.0.8</AspNetCoreVersion8>
+    <EfCoreVersion8>8.0.8</EfCoreVersion8>
+    <RuntimeVersion9>9.0.0-preview.7.24405.7</RuntimeVersion9>
+    <AspNetCoreVersion9>9.0.0-preview.7.24406.2</AspNetCoreVersion9>
+    <EfCoreVersion9>9.0.0-preview.7.24405.3</EfCoreVersion9>
   </PropertyGroup>
+  <!-- Independent from TargetFrameWorks-->
```

Comment:
Can you add a line of spacing before these “global” comments to have a cleaner file?